src/System.ComponentModel.Composition.Registration/pkg/System.ComponentModel.Composition.Registration.pkgproj

```diff
@@ -2,9 +2,10 @@
     <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
     <ItemGroup>
     <ProjectReference Include="..\ref\System.ComponentModel.Composition.Registration.csproj">
-      <SupportedFramework>uap10.0.16299;netcoreapp2.0;net45;$(AllXamarinFrameworks)</SupportedFramework>
+      <SupportedFramework>netcoreapp3.0;net45;$(AllXamarinFrameworks)</SupportedFramework>
     </ProjectReference>
     <ProjectReference Include="..\src\System.ComponentModel.Composition.Registration.csproj" />
+    <InboxOnTargetFramework Include="$(AllXamarinFrameworks)" />
```

Comment:
If you want this to be inbox on Xamarin you have to record it as such in the packageIndex.  This is what is causing your build failures.  Usually when we do this we get an ACK from @marek-safar, @mhutch, or @akoeplinger.  Alternatively we can let the package deliver its netstandard2.1 implementation here, Xamarin will catch up and implement ns2.1 with the TypeInfo constructor.

Reply:
Spoke with @marek-safar offline and we agreed that it's not worth marking them inbox for Xamarin.